ProjectLighthouse/Controllers/EnterLevelController.cs

```diff
@@ -118,6 +119,7 @@ public async Task<IActionResult> EnterLevel(int id)
             {
                 v = await visited.FirstOrDefaultAsync();
             }
+            if (v == null) return StatusCode(403, "");
 
             if (v == null)
             {
```

Comment:
Not sure why this is being checked and returned twice.

Reply:
thought I pushed the commit to fix this, fixed in https://github.com/LBPUnion/project-lighthouse/pull/62/commits/a7443c16a0e240b50b4026f98ad60d5e744fc6a5